lasagne/tests/layers/test_dense.py

```diff
@@ -113,3 +115,173 @@ def test_named_layer_param_names(self, DenseLayer, dummy_input_layer):
 
         assert layer.W.name == "foo.W"
         assert layer.b.name == "foo.b"
+
+
+class TestNonlinearityLayer:
+    @pytest.fixture
+    def NonlinearityLayer(self):
+        from lasagne.layers.dense import NonlinearityLayer
+        return NonlinearityLayer
+
+    @pytest.fixture
+    def layer_vars(self, dummy_input_layer):
+        from lasagne.layers.dense import NonlinearityLayer
```

Comment:
Not critical at all, but you could use the fixture here instead of importing it again. (Same for `NINLayer` below.)